keras/optimizers/optimizer_experimental/adamw.py

```diff
@@ -136,6 +136,7 @@ def build(self, var_list):
     if hasattr(self, '_built') and self._built:
       return
     self._built = True
+    self._exclude_from_weight_decay = exclude_from_weight_decay
```

Comment:
`exclude_from_weight_decay or []` (need to support the default `None` case)

Reply:
For the future you can define it as `exclude_from_weight_decay: Optional[List[str]] = None` and I added the None case solution.